### DISCUSSION TITLE
Request for Copernicus marine high resolution coastal geophys datasets

### DISCUSSION THREAD

**Discussion by datasplod** [microsoft/PlanetaryComputer#57] [Data request]
I'd like to request the high resolution coastal optics and geophysical datasets provided by Copernicus Marine Service.  This series of datasets provides turbidity, solid particulate matter concentration, particulate backscattering and chlorophyll-a for the coasal area derived from Sentinel2 at 100m resolution for a 20km coastal zone.  The products are provided seperately for 6 regions: Arctic sea, Baltic sea, North western shelf, Iberian Sea, Mediterranean Sea, Black Sea.  Request for L4 product monthly means.

These are high resolution coastal datasets so pretty unique in the quality they provide for the coastal zone that can be used for the purposes of coastal research, managing marine resources, coastal monitoring, deriving coastal environmental parameters, climate and seasonal studies.  

The datasets are at the following links:
North-west shelf region:  [https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_NWS_BGC_HR_L4_NRT_009_209/INFORMATION](https://www.google.com/url?q=https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_NWS_BGC_HR_L4_NRT_009_209/INFORMATION&sa=D&source=buganizer&usg=AOvVaw1hPIPRaJMiZXwtq3sVUzqE)
Baltic sea:  [https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_BAL_BGC_HR_L4_NRT_009_208/INFORMATION](https://www.google.com/url?q=https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_BAL_BGC_HR_L4_NRT_009_208/INFORMATION&sa=D&source=buganizer&usg=AOvVaw1pSAQlKO9XU-DRxNPHEOyH)
Mediterranean sea:  [https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_MED_BGC_HR_L4_NRT_009_211/INFORMATION](https://www.google.com/url?q=https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_MED_BGC_HR_L4_NRT_009_211/INFORMATION&sa=D&source=buganizer&usg=AOvVaw3w2HTZmKuRjZLWknvWzA-a)
Iberian Sea:  [https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_IBI_BGC_HR_L4_NRT_009_210/INFORMATION](https://www.google.com/url?q=https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_IBI_BGC_HR_L4_NRT_009_210/INFORMATION&sa=D&source=buganizer&usg=AOvVaw2QTcld4HyTqsqKTO6V9uEK)
Black sea:  [https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_BLK_BGC_HR_L4_NRT_009_212/INFORMATION](https://www.google.com/url?q=https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_BLK_BGC_HR_L4_NRT_009_212/INFORMATION&sa=D&source=buganizer&usg=AOvVaw1RTxKCJHkgAeuhu-j-aPko)
Arctic sea:  [https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_ARC_BGC_HR_L4_NRT_009_207/INFORMATION](https://www.google.com/url?q=https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_ARC_BGC_HR_L4_NRT_009_207/INFORMATION&sa=D&source=buganizer&usg=AOvVaw15byTK0zwnH1Y9gxN_H0Xr)

And to download the data:
North-west shelf region:  [https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_NWS_BGC_HR_L4_NRT_009_209/DATA-ACCESS](https://www.google.com/url?q=https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_NWS_BGC_HR_L4_NRT_009_209/DATA-ACCESS&sa=D&source=buganizer&usg=AOvVaw2zCzrRAICgTf6QvSIbIobd)
Baltic sea:  [https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_BAL_BGC_HR_L4_NRT_009_208/DATA-ACCESS](https://www.google.com/url?q=https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_BAL_BGC_HR_L4_NRT_009_208/DATA-ACCESS&sa=D&source=buganizer&usg=AOvVaw3fwxvfCWC5fh8RzbWgoTRV)
Mediterranean sea:  [https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_MED_BGC_HR_L4_NRT_009_211/DATA-ACCESS](https://www.google.com/url?q=https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_MED_BGC_HR_L4_NRT_009_211/DATA-ACCESS&sa=D&source=buganizer&usg=AOvVaw1qCbAkvi1q7CUAatCIIdOg)
Iberian Sea:  [https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_IBI_BGC_HR_L4_NRT_009_210/DATA-ACCESS](https://www.google.com/url?q=https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_IBI_BGC_HR_L4_NRT_009_210/DATA-ACCESS&sa=D&source=buganizer&usg=AOvVaw2_SLWtRVnpIIcOZnOzz6zb)
Black sea: [https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_BLK_BGC_HR_L4_NRT_009_212/DATA-ACCESS](https://www.google.com/url?q=https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_BLK_BGC_HR_L4_NRT_009_212/DATA-ACCESS&sa=D&source=buganizer&usg=AOvVaw0T-aAKp1JBm_62z8BTq8az)
Arctic sea:  [https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_ARC_BGC_HR_L4_NRT_009_207/DATA-ACCESS](https://www.google.com/url?q=https://resources.marine.copernicus.eu/product-detail/OCEANCOLOUR_ARC_BGC_HR_L4_NRT_009_207/DATA-ACCESS&sa=D&source=buganizer&usg=AOvVaw2be1YtLah5YEdbioGLcu5y)

**Comment by TomAugspurger** [comment]
Thanks for submitting this request.

Given that data is distributed in NetCDF, we'd likely use pangeo-forge to process the data to cloud-optimized format. We can add this to the data backlog, but in the meantime https://pangeo-forge.readthedocs.io/en/latest/ might be worth looking at if you want to try processing it yourself.
